lib/exec.js

```diff
@@ -41,13 +41,24 @@ function execute(command, args, options) {
 
     var okToCache = true;
     logger.debug({type: "executing", command: command, args: args});
-    // AP: Run this process in specified tmpDir. tmpDir on Windows volume is required for WSL execution.
+    // AP: Run Windows-volume executables in winTmp. Otherwise, run in tmpDir (which may be undefined).
     // https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options
-    var child = child_process.spawn(command, args, {
-        cwd: process.env.tmpDir,
-        env: env,
-        detached: process.platform === 'linux'
-    });
+    var child;
+    if (command.startsWith("/mnt") && process.env.wsl) {
```

Comment:
I think maybe a ternary operator for cwd instead of duplicating the rest of the options is cleaner solution here?

Reply:
I didn't know you could put a ternary in there :) Will do.